cms/static/js/views/assets.js

```diff
@@ -20,6 +20,32 @@ define([
              ModalUtils, ViewUtils, NotificationView, asset_library_template) {
         var CONVERSION_FACTOR_MBS_TO_BYTES = 1000 * 1000;
 
+        $('.clearable').on('keyup focus change', function(e){
+            if($(this).val().length > 0){
+               $(this).addClass('x');
+            }else{
+               $(this).removeClass('x');
+            }
+        }).on('click', function(e){
+            if(this.offsetWidth-18 < e.clientX-this.getBoundingClientRect().left){
+                e.preventDefault();
+                $(this).removeClass('x').val('').change();
+                $('.filter-criteria').val('');
+                $('.filter-criteria-form').submit();
+            }
+        }).on('mousemove', function( e ){
+            if(this.offsetWidth-18 < e.clientX-this.getBoundingClientRect().left){
+                $(this).css('cursor', 'pointer');
+            }else{
+                $(this).css('cursor', 'text');
+            }
+        });
+
+        $(window).ready()
+        {
+            $('.clearable').change();
+        }
+
```

Comment:
@jonathan-byrne sorry, its difficult to mark the right part of the code. No, the syntax I mean in the callback itself. Should look something like:
```
$(window).ready(function(){
    $('.clearable').change();
}
```
Otherwise, I think its just calling the ready and the change as separate functions.